.github/workflows/deploy-paas.yml

```
@@ -0,0 +1,52 @@
+name: Deploy publish app to PAAS
+
+on:
+  push:
+    branches:
+      - master
+
+
+env:
+  TF_VAR_api_url: https://api.london.cloud.service.gov.uk
+  TF_VAR_user: ${{ secrets.TF_VAR_user }}
+  TF_VAR_password: ${{ secrets.TF_VAR_password }}
+  TF_VAR_SECRET_KEY_BASE: ${{ secrets.TF_VAR_SECRET_KEY_BASE }}
+  TF_VAR_SENTRY_DSN: ${{ secrets.TF_VAR_SENTRY_DSN }}
+  TF_VAR_SETTINGS__GOOGLE__GCP_API_KEY: ${{ secrets.TF_VAR_SETTINGS__GOOGLE__GCP_API_KEY }}
+  TF_VAR_SETTINGS__GOOGLE__MAPS_API_KEY: ${{ secrets.TF_VAR_SETTINGS__GOOGLE__MAPS_API_KEY }}
+  BACKEND_STORAGE_ACCOUNT_NAME: s121d01tfstatestr
+  BACKEND_ACCESS_KEY: ${{ secrets.BACKEND_ACCESS_KEY }}
+  BACKEND_KEY: paas-ptt.tfstate
+  TERRAFORM_VAR_FILE: terraform_qa.tfvars
+  PAAS_TF_DIR: $HOME/work/publish-teacher-training/publish-teacher-training/terraform/paas
+  CF_PROVIDER_DIR: $HOME/.terraform.d/plugins/linux_amd64/terraform-provider-cloudfoundry
+  TF_VAR_SETTINGS_DFE_SIGNIN_SECRET: ${{ secrets.TF_VAR_SETTINGS__DFE_SIGNIN__SECRET }}
+  TF_VAR_SETTINGS_LOGSTASH_HOST: ${{ secrets.TF_VAR_SETTINGS__LOGSTASH__HOST }}
```

Should these look more like the final form of the variable?

```suggestion
  TF_VAR_SETTINGS__LOGSTASH__HOST: ${{ secrets.TF_VAR_SETTINGS__LOGSTASH__HOST }}
```

---

amended